_content/essays/great-stake-to-the-heart_chokyi-lodro.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Great Stake to the Heart: A Song of Sadness"
+authors:
+  - chokyi-lodro
+translator: "Adam Pearcey"
+external_url: "https://www.lotsawahouse.org/tibetan-masters/jamyang-khyentse-chokyi-lodro/great-stake-to-heart-song"
+drive_links:
+  - "https://drive.google.com/file/d/1rdRhCMtvI5C6Vn4cN8x4i93YnxjyqdPg/view?usp=sharing"
+course: problems
+tags:
+  - refuge
+  - lamentation
+  - guru
```

Comment:
`guru-worship` yeah

Reply:
Changed it. I was going to use it at the start, but it felt funny though since guru-worship seems to have a negative connotation - maybe that's just my modern projection.